charts/openshift-metering/templates/reporting-operator/reporting-operator-deployment.yaml

```diff
@@ -441,6 +445,11 @@ spec:
 {{- end }}
 {{- end }}{{/* end of authProxy.enabled */}}
       volumes:
+{{- if and .Values.networking.useGlobalProxyNetworking .Values.networking.proxy.config.trusted_ca_bundle }}
+        - name: proxy-user-ca-bundle
+          configMap:
+            name: user-ca-bundle
```

Comment:
We most likely don't want the ConfigMap we're referencing to also match the `user-ca-bundle` ConfigMap in the `openshift-config` namespace. Let's add something like a `reporting-operator-*` or `metering-operator-*` prefix to distinguish between the two.